bld/CLMBuildNamelist.pm

```diff
@@ -3917,15 +3919,19 @@ sub setup_logic_lai_streams {
        }
      }
   } else {
-     # If bgc is CN/CNDV then make sure none of the LAI settings are set
-     if ( defined($nl->get_value('stream_year_first_lai')) ||
-          defined($nl->get_value('stream_year_last_lai'))  ||
-          defined($nl->get_value('model_year_align_lai'))  ||
-          defined($nl->get_value('lai_tintalgo'        ))  ||
+     # If bgc is BGC/BGCDV then make sure none of the LAI settings are set
+     if ( &value_is_true($nl->get_value('use_lai_streams'))) {
+        $log->fatal_error("When bgc is NOT SP use_lai_streams cannot be .true.\n" .
```

Comment:
I think the error should be slightly changed to point out SP or FATES-SP.

Reply:
It would be good to add some build-namelist fail test around the new fatal_errors...